### ISSUE TITLE
Bengali language is not properly formatted

### ISSUE THREAD

**Issue by moshiurse** [jwasham/coding-interview-university#905]
Hello @jwasham 
I have seen [https://github.com/jwasham/coding-interview-university/blob/main/translations/README-bn.md](this Readme) File of Bangla/Bengali language. It's properly translated I think but no properly formatted. Can I edit this and send you a pull request? (Not sure about the procedure). 
Thanks in advance.

**Comment by jwasham** [comment]
I think you can fork and then do a pull request.

**Comment by moshiurse** [comment]
Okay, Thank You. I will be working on this

**Comment by jwasham** [comment]
Any ideas?

**Comment by moshiurse** [comment]
Is it okay to send my PR by doing part by part.
I started working on this then stopped after some days because of some personal business.

**Comment by jwasham** [comment]
Of course, you can open a PR and add to it over time. I won't merge it in until you're ready.

**Comment by moshiurse** [comment]
Great. Thanks a lot
